README.md

```diff
@@ -72,7 +72,6 @@ sudo ${HOMEBREW_PREFIX}/opt/socket_vmnet/bin/socket_vmnet --vmnet-gateway=192.16
 
 To install the launchd service:
 ```bash
-brew tap homebrew/services
 # sudo is necessary for the next line
 sudo ${HOMEBREW_PREFIX}/bin/brew services start socket_vmnet
```

Comment:
We have this warning:

> **Warning**
> Typically, the `socket_vmnet` binary in the `${HOMEBREW_PREFIX}` can be replaced by any user in the `admin` group.

I think the issue is worse. The socket_vmnet executable can be replaced by a program *you* run. Some program or script that you run as your user (without sudo) can replace socket_vmnet executable. Then when it is run by lima, the replaced executable run as root.

So the real fix is to remove the brew service instructions, they are unsafe.

If we want to give instructions, they must include all the commands to make running the brew version safe:

- Change ownership (recursively) of the directory where socket_vmnet is installed
- Create safe directory for the socket and logs
- Fixing ownership to make brew upgrade work

With these changes the brew "easy" solution is not so attractive.

I don't think this can be fixed without installing the files as root, like other package managers.

Reply:
> I don't think this can be fixed without installing the files as root, like other package managers.

I think this is correct. For that reason we used to require that the daemon files are owned by `root`. But that requirement has been softened in #1220 for the benefit of Homebrew users.

It creates a vulnerability that allows programs run by the admin user to escalate to root privileges without getting a sudo prompt.

So installing `socket_vmnet` with brew is not a good idea if you would have qualms about enabling password-less `sudo`.